net-misc/httpie/httpie-2.2.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
Can you add Python 3.9 if tests pass?

Reply:
Upstream docs state that Python >= 3.6 should work, but I have not yet tested it. It is however on my to-do list.